config/spryk/spryks/Spryker/Glue/AddGlueResourceMethodResponse.yml

```diff
@@ -146,21 +146,21 @@ postSpryks:
     # What are the client methods we would need to bridge??
 
     # Add the controller method to handle the request
-    - AddGlueBackendApiControllerMethodGet:
+    - AddGlueApiApplicationControllerMethodGet:
           condition: "applicationType === 'Backend' && httpMethod === 'get' && isBulk === false"
```

Comment:
From the condition (you do not change it) I take it that there is (or will be) a similar command/directive for storefront. Is this correct?